### ISSUE TITLE
Moonlight generator : can't use keyboard and mouse anymore

### ISSUE THREAD

**Issue by substring** [recalbox/recalbox-configgen#41]
As we are forcing input devices with the generator, we just map joysticks, no keyboard nor mouse

**Comment by plugpoint** [comment]
Is there any way to map the keyboard and mouse manually?

**Comment by substring** [comment]
nope. But sooner or later i'll have to correct that in configgen

**Comment by substring** [comment]
@plugpoint just found a way to do it easily : add some `input = /dev/input/event0` to `/recalbox/share/system/configs/moonlight/moonlight.conf`

You may need 2 lines : 1 for the keyboard event, one for the mouse event

**Comment by plugpoint** [comment]
Thanks it worked

i added these lines:

input = /dev/input/event0

input = /dev/input/event2


event1 didnt seem to work

On a side note, I also noticed that when 5 controllers are plugged in, moonlight doesnt start(this is without keyboard and mouse).

**Comment by substring** [comment]
Yeah, that's the downside i wrote in my comment : you can force only 4 input devs

**Comment by plugpoint** [comment]
For information's sake I just ran a few tests.
I can get 4 controllers + keyboard + mouse running on moonlight steam.

**Comment by substring** [comment]
Hey great ! Dunno why it was crashing before mentionning too many inputs

**Comment by substring** [comment]
@plugpoint what have you done to make that work ? any special step ? or it just crashes at 1 and 3 controllers + moue/keyboard ?

**Comment by plugpoint** [comment]
I think its because i only configure 2 controllers on the recalbox interface and only set p1 and p2 for the configured controllers.

**Comment by myles786** [comment]
This worked for me!

**Comment by Frostoman** [comment]
Hi everyone. I tried many combinations of "input = /dev/input/event0,1 and 2" in "moonlight.conf", and all what I got is:
- keyboard and mouse working, but not gamepad
- gamepad working, but not keyboard and mouse
- keyboard, mouse and gamepad not working

Can sombody help me configure Moonlight to use keyboard, mouse and gamepad at same time?
